docs/protocol-levers.md

```diff
@@ -1,5 +1,7 @@
 # Protocol levers
 
+**WARNING**: this doc is outdated after Lido V2 upgrade. For the more up-to-date version see https://docs.lido.fi/deployed-contracts/
```

Comment:
the link is wrong, I guess
should be https://docs.lido.fi/guides/protocol-levers

Reply:
hmm, removed the protocol-levers.md at all and updated link in README. Does it seem better for you?

Reply:
yeah, ty!